enums/enumgen/enumgen.go

```diff
@@ -47,13 +46,13 @@ func Generate(cfg *Config) error { //types:add
 	pkgs, err := ParsePackages(cfg)
 	if err != nil {
 		if logx.UserLevel <= slog.LevelInfo {
-			errors.Log(err)
+			logx.PrintlnInfo(err)
```

Comment:
The whole point of `logx.PrintlnInfo` is that you **don't** have to do the if statement on log level.